### PR TITLE
Repair: Set URI

### PR DESCRIPTION
**Pull request by dokshina**
- [x] Tests
- [x] Documentation (see #343)
- [x] Changelog

First part of #327

### CODE REVIEW

File: cli/common/files.go [hunk 1]
```diff
@@ -84,20 +84,31 @@ func FileLinesScanner(file *os.File) *bufio.Scanner {
 	return scanner
 }
 
-// GetFileContent returns file content
-func GetFileContent(path string) (string, error) {
+// GetFileContent returns file content bytes
```
Comment (olegrok) [inline cli/common/files.go:87]: _as bytes slice_

File: cli/common/files.go [hunk 2]
```diff
@@ -84,20 +84,31 @@ func FileLinesScanner(file *os.File) *bufio.Scanner {
...
 	fileContent, err := ioutil.ReadAll(file)
+	if err != nil {
+		return nil, err
+	}
+
+	return fileContent, nil
+
+}
+
+// GetFileContent returns file content
```
Comment (olegrok) [inline cli/common/files.go:104]: _as a string_

File: cli/repair/common.go [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+		return nil, fmt.Errorf("%s is not a directory", ctx.Running.DataDir)
+	}
+
+	if ctx.Cli.Verbose {
+		log.Infof("Data directory is set to: %s", ctx.Running.DataDir)
+	}
+
+	workDirs, err := ioutil.ReadDir(ctx.Running.DataDir)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to list data directory: %s", err)
```
Comment (olegrok) [inline cli/repair/common.go:36]: `list data dir` -> `list a data dir`?

File: cli/repair/common.go [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+			changelog = append(changelog, "Topology config changes:")
+			changelog = append(changelog, configDiff...)
+		} else {
+			changelog = append(changelog, "Topology config wasn't changed")
+		}
+
+		changelog = append(changelog, "") // an empty line to separate instances changes
+	}
+
+	if !ctx.Repair.DryRun {
```
Comment (olegrok) [inline cli/repair/common.go:134]: Maybe running without "dry-run" option is more common case than without it.

Consider

```go
if ctx.Repair.DryRun {
    return changelog, nil
}
-- actions to run without dry-run
```

But it's mostly up to you

File: cli/repair/repair.go [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+			resCh <- res
+		}(workDirPath, workDirName, resCh)
+	}
+
+	var errors []error
+
+	for i := 0; i < len(appWorkDirNames); i++ {
+		select {
+		case res := <-resCh:
```
Comment (olegrok) [inline cli/repair/repair.go:56]: Please add some timeout

File: cli/repair/common.go (hunk 4 above)
Comment (dokshina) [reply cli/repair/common.go:134]: Yes, I agree.

File: cli/repair/common.go (hunk 3 above)
Comment (dokshina) [reply cli/repair/common.go:36]: Maybe `the`?

Comment (dokshina) [reply cli/repair/common.go:36]: Changed to `the`.

**Pull request merged**